sqlcommenter/taggers.go

```diff
@@ -0,0 +1,59 @@
+package sqlcommenter
+
+import (
+	"context"
+	"fmt"
+	"runtime/debug"
+)
+
+type DriverVersionTagger struct {
+	version string
+}
+
+func NewDriverVersionTagger() DriverVersionTagger {
+	info, ok := debug.ReadBuildInfo()
+	ver := "ent"
+	if ok {
+		ver = fmt.Sprintf("ent:%s", info.Main.Version)
+	}
+	return DriverVersionTagger{ver}
+}
```

Comment:
Why it's not an option, like `WithVersion()`? Also, you're reading the main module.